scripts/do-release.sh

```
@@ -96,6 +97,7 @@ function create_pr() {
         return 1
     fi
 
+    to_review=$(echo "${to_review}" | dryrun grep "http.*")
```

Is the goal to extract the URL, or to fail if there’s no URL?

---

yes and yes, I want to extract the URL but preferably (when not in "dryrun" mode) it should fail if there's no URL since that's unexpected at this stage (URL is also used for auto merge)

mind you it won't stop the script immediately, these failure are collected and reported in the end in a batch